src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -5,6 +5,7 @@
 
 #include <vector>
 
+#include "Domain/Creators/DomainCreator.hpp"
```

Comment:
Shouldn't this include (and in all the other executables) be somewhere else?

Reply:
I guess in a way it could be in `Domain/OptionsTags.hpp` but then it gets sucked in everywhere again. I would say `ParallelAlgorithms/DG/InitializeDomain.hpp` is reasonable. Thoughts?

Reply:
Actually, it's `DgElementArray.hpp` that adds it to the cache. I'll add the includes there.